src/resolvers/line_items.ts

```diff
@@ -256,22 +252,12 @@ async function getUpdateAttributes(
   return updateAttributes
 }
 
-async function updateGroupIdAttr(
-  lineItem: LineItem,
+async function getUpdateGroupIdAttr(
   input: Static<typeof updateLineItemInput>['input'],
-  updateAttributes: Partial<LineItemAttributes>,
   attr: 'acceptedReceivingGroupId' | 'proposedReceivingGroupId',
 ) {
-  if (
-    !('acceptedReceivingGroupId' in input) ||
-    input[attr] === lineItem[attr]
-  ) {
-    return
-  }
-
```

Comment:
I removed this because if the value stayed the same, it would get deleted in the database, which isn't what we wanted.

Reply:
This is the fix for #216.